cli/azd/pkg/input/console.go

```diff
@@ -407,6 +408,9 @@ func (c *AskerConsole) ShowSpinner(ctx context.Context, title string, format Spi
 		// calling Start may result in an additional line of output being written in non-tty scenarios
 		_ = c.spinner.Start()
 	}
+
+	watchTerminalInterrupt(c)
```

Comment:
I suspect this ends up leaking a goroutine each time the spinner starts. I'd like to propose a quick change here to avoid this behavior -- when I initially wrote some of this, it was intentionally coded to only have a single signal interception, we may need to change this behavior slighty to match